src/DDTrace/Integrations/Symfony/V4/SymfonyBundle.php

```diff
@@ -48,13 +52,18 @@ public function boot()
 
         // Create a span that starts from when Symfony first boots
         $scope = $tracer->startActiveSpan('symfony.request');
-        $symfony_request_span = $scope->getSpan();
-        $symfony_request_span->setTag(Tags\SERVICE_NAME, $this->getAppName());
-        $symfony_request_span->setTag(Tags\SPAN_TYPE, Types\WEB_SERVLET);
+        $symfonyRequestSpan = $scope->getSpan();
+        $symfonyRequestSpan->setTag(Tags\SERVICE_NAME, $this->getAppName());
+        $symfonyRequestSpan->setTag(Tags\SPAN_TYPE, Types\WEB_SERVLET);
 
         // public function handle(Request $request, $type = HttpKernelInterface::MASTER_REQUEST, $catch = true)
-        dd_trace(HttpKernel::class, 'handle', function ($request, ...$args) use ($symfony_request_span) {
+        dd_trace(HttpKernel::class, 'handle', function ($request, ...$args) use ($symfonyRequestSpan) {
             $scope = GlobalTracer::get()->startActiveSpan('symfony.kernel.handle');
+            $symfonyRequestSpan->setTag('request.method', $request->getMethod());
+            $symfonyRequestSpan->setTag('request.uri', $request->getUri());
+            $symfonyRequestSpan->setTag('request.content-type', $request->getContentType());
+            $symfonyRequestSpan->setTag('request.referrer', $request->headers->get('referer'));
+            $symfonyRequestSpan->setTag('request.client-ip', $request->getClientIp());
```

Comment:
We would prefer to not store the client IP for now, mainly for privacy reasons. We can consider adding hooks, soon, to let users modify the request spans as they like, but for now please remove this line.

Reply:
Fair enough, I'll remove it